src/multirust-cli/rustup_mode.rs

```diff
@@ -160,6 +162,13 @@ pub fn cli() -> App<'static, 'static> {
                      .short("y")))
             .subcommand(SubCommand::with_name("upgrade-data")
                 .about("Upgrade the internal data format.")))
+        .subcommand(SubCommand::with_name("telemetry")
+                .about("Enable or disable rust telemetry")
+                .arg(Arg::with_name("enabled")
+                        .takes_value(true)
+                        .value_name("telemetry")
+                        .help("Set telemetry 'on' or 'off'")
+                        .possible_values(&["on", "off"])))
```

Comment:
Let's set this to `hidden(true)` for the time being until we've fleshed out the feature a bit more and really want people to turn it on.